h/httpserver.h

```diff
@@ -146,6 +146,7 @@ typedef int HttpServiceServe(struct HttpService_tag *service, HttpResponse *resp
 typedef int AuthExtract(struct HttpService_tag *service, HttpRequest *request);
 typedef int AuthValidate(struct HttpService_tag *service, HttpRequest *request);
 typedef int HttpServiceInsertCustomHeaders(struct HttpService_tag *service, HttpResponse *response);
+typedef int AuthorizationHandler(struct HttpService_tag *service, HttpRequest *request, HttpResponse *response, void *userData);
```

Comment:
Can we prepend this typedef with `Http` to avoid name clashes. The reason is, `AuthorizationHandler` seems to be too generic. So perhaps, `HttpAuthorize` (the rest of the typedefs are verbs)?